<launch>
	<node name="sync"  pkg="serial_node" type="serial_node_node"  output="screen">
		<param name="serial_port" type="string" value="/dev/ttyUSB0"/>
		<param name="serial_baudrate"  type="int" value="115200"/>
		<param name="frame_id" type="string" value="agribot"/>
	</node>
</launch>
